builders/xcm/remote-evm-calls.md

```diff
@@ -0,0 +1,337 @@
+
+As explained in the [Introduction](#introduction), the paths that a regular and remote EVM call takes to get to the EVM are quite different. The main reason behind this difference is the dispatcher of the transaction.
+
+A regular EVM call has an apparent sender who signed the Ethereum transaction. With remote EVM calls, the signer signed an XCM transaction in another chain. Moonbeam receives that XCM message which must be constructed with the following instructions:
+
+ - [`Descend Origin`](https://github.com/paritytech/xcm-format#descendorigin){target=_blank}
+ - [`WithdrawAsset`](https://github.com/paritytech/xcm-format#withdrawasset){target=_blank} 
+ - [`BuyExecution`](https://github.com/paritytech/xcm-format#buyexecution){target=_blank} 
+ - [`Transact`](https://github.com/paritytech/xcm-format#transact){target=_blank} 
+
+The first instruction, `DescendOrigin`, will mutate the origin of the XCM call on the Moonbeam side to a keyless account through the `Multilocation-derivative account` mechanism described in the [Relevant XCM Definitions section](#general-xcm-definitions). The remote EVM call is dispatched from that keyless account (or a related [proxy](/tokens/manage/proxy-accounts/){target=_blank}), therefore this transaction is not signed (it does not have the real `v-r-s` values of the signature, but `0x1` instead).
+
+Because a remote EVM call does not have the actual `v-r-s` values of the signature, there could be collusion problems of the EVM transaction hash, as it is calculated as the Keccak256 hash of the signed transaction blob. Consequently, if two accounts with the same nonce submit the same transaction object, they will end up with the same EVM transaction hash. Therefore, all remote EVM transactions use a global nonce that is attached to the [Ethereum-XCM pallet](https://github.com/PureStake/moonbeam/tree/master/pallets/ethereum-xcm){target=_blank}.
+
+Another significant difference is in terms of the gas price. The fee for remote EVM calls is charged at an XCM execution level. Consequently, the EVM gas price is zero, and the EVM will not charge for the execution itself. This can also be seen in the receipt of a remote EVM call transaction.
+
+Lastly, a lower gas limit is available for remote EVM calls. Currently, an XCM message can only take `20,000,000,000` weight (that is, `0.02` seconds of block execution time). For all Moonbeam-based networks, there is a ratio of [`25,000` units of gas per unit of weight](https://github.com/PureStake/moonbeam/blob/master/runtime/moonbase/src/lib.rs#L371-L375){target=_blank}. Considering that you need some of the XCM message weight limit to execute the XCM instructions themselves, a remote EVM call might have around `18,000,000,000` weight left, which is `720,000` gas units. Consequently, the maximum gas limit you can provide for a remote EVM call is around `720,000` gas units. Note that this might change in the future.
```

Comment:
Consider making the separation of gas and weight more apparent

Main points:
1. Weight is what polkadot uses to pay for fees.
2. When executing an XCM message, there is a maximum weight.
3. Some weight is reserved for mandatory XCM instruction execution
4. The rest of the weight can be used for the EVM transaction, which equals roughly 720,000 gas units